### PR TITLE
docs: Provide worked Kafka example with beam

### PR DESCRIPTION
**Pull request by dpcollins-google**
This is intended to be a worked example: It may or may not be submitted or could be moved depending on tianzi's thoughts.

### CODE REVIEW

File: samples/snippets/src/main/java/pubsublite/kafka/KafkaIOReadExample.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+package com.google.cloud.pubsublite.demo;
```
Comment (lesv) [inline samples/snippets/src/main/java/pubsublite/kafka/KafkaIOReadExample.java:1]: Shouldn't use `com.google.*` for sample / demo package names.  `com.example.*` or `productname.xxx`

Comment (lesv) [reply samples/snippets/src/main/java/pubsublite/kafka/KafkaIOReadExample.java:1]: We don't want customers accidently publishing w/ com.google

**Review by lesv — CHANGES_REQUESTED** [review]
I'll come back to this after my next meeting, I don't see a pom.xml, so I wonder a bit.  Also, it looks like there might be a line length issue.

**Comment by lesv** [comment]
Ah - I see checkstyle has failed, please fix those issues and then ping me.

**Comment by dpcollins-google** [comment]
Closing this, it was never intended to be merged.

**Pull request closed without merging**